documentation/api/api-versioning.md

```diff
@@ -0,0 +1,56 @@
+# Overview
+
+This document will detail how we handle versioning our API right now, as well a long-term idea that we will want to circle back to.
+
+## Why versioning?
```

Comment:
Glad that you included this section! Always helps to ground our approach in "why"